src/kernel/coinstats.h

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_KERNEL_COINSTATS_H
```

Comment:
Note to other reviewers, the `node/coinstats.h` is deleted in subsequent commits, so the fact that they share a filename isn't really interesting.